crates/ruff/resources/test/fixtures/pylint/else_if_used.py

```diff
@@ -0,0 +1,47 @@
+"""
+Test for else-if-used
+"""
+def ok0():
+    """Should not trigger on elif"""
+    if 1:
```

Comment:
It looks like this is being flagged as an error in the fixture, when it should be considered ok.